lumicks/pylake/detail/widefield.py

```diff
@@ -129,7 +132,9 @@ def num_frames(self):
 
 
 class ImageDescription:
-    def __init__(self, src):
+    def __init__(self, src, align):
+        self.is_rgb = src.tags["SamplesPerPixel"].value == 3
+        self._align = align
```

Comment:
I wonder if `_want_align` may be a better name, since we don't know at this point whether alignment is even possible (there's no check for the presence of alignment matrices yet).

What I'd otherwise find a bit confusing is for example a single color image, where this property is `True` and a three color image without alignment matrices where this property is also `True` even though it can clearly not be aligned.

Reply:
indeed, especially now that we support reading of images that were previously aligned and then exported from pylake, we actually can't fully specify the state with a simple boolean. Additonally, yes the warnings can be confusing because they don't necessarily match the state. What we really need is something like `align_requested : bool` and `alignment_status : enum` where the status could be `ready`, `already applied`, `missing` etc ... 

I played around with this a bit, but it's actually quite a bit of additions/refactoring that needs to be done. I'd like to open this up as a 2nd PR based on this one

Reply:
Alright. Sounds fair.